extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -176,29 +179,31 @@ List<MessageBundleBuildItem> processBundles(BeanArchiveIndexBuildItem beanArchiv
 
                     // Find localized files
                     Map<String, Path> localeToFile = new HashMap<>();
+                    // Merge candidates are localized files that are duplicate to localized interface
```

Comment:
Hm, WRT "Merge candidates... are duplicate" I think that those localized files are rather _supplementary_ or _additional_.